style/item.css

```diff
@@ -177,3 +177,26 @@
   resize: none;
   height: 215px;
 }
+
```

Comment:
The features aren't aligning properly, it seems. I'll have a suggestion for some changes to the layout/styling, hopefully tonight.

Reply:
I pushed a commit making the section more compact. See if you like it and make some changes if you don't:)